backend/foodbank_southlondon/api/events/views.py

```diff
@@ -29,8 +31,8 @@ def get(self) -> Tuple[pd.DataFrame, int, int]:
         """List all Events."""
         params = parsers.events_params.parse_args(flask.request)
         refresh_cache = params["refresh_cache"]
-        request_ids = set(request_id.strip() for request_id in (params["request_ids"] or ()))
-        event_names = set(event_name.strip() for event_name in (params["event_names"] or ()))
+        request_ids = set(request_id.strip() for request_id in flatten((params["request_ids"] or ())))
```

Comment:
I was getting an error "strip is not a member of list" when testing. I gues we've never tested multiple values for these args before? They're passed as a list of lists

Reply:
Why are they / what is passing them as, a list of lists?

Reply:
I think I was expecting them to be a comma-separated list of values.
https://flask-restx.readthedocs.io/en/latest/parsing.html?highlight=action%3Dsplit#multiple-values-lists

Reply:
Ah good point, the new request in `_process_warnings` needed `".".join(request_ids)`. I've changed it and removed the flatten